src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -4039,4 +4039,25 @@ inline RppStatus compute_packed_to_planar_host(T* srcPtr, RppiSize srcSize, T* d
     return RPP_SUCCESS;
 }
 
+inline RppStatus compute_cmn_24_host(__m256 &pVecR, __m256 &pVecG, __m256 &pVecB, __m256 *pCMNParams)
+{
+    pVecR = _mm256_sub_ps(pVecR, pCMNParams[0]);
+    pVecG = _mm256_sub_ps(pVecG, pCMNParams[0]);
+    pVecB = _mm256_sub_ps(pVecB, pCMNParams[0]);
+    
```

Comment:
Ensure you set the "Trim Trailing Whitespace while saving a file" setting in VS code to on.